docs/DeveloperGuide.md

```diff
@@ -941,19 +930,30 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Adding a patient
+
+1. Adding a new employee to the active employee directory
+
+   1. Test case: `c-add n/John Doe p/98765432 e/87654321 a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney`<br>
```

Comment:
Maybe can use "t/friday t/monday" instead of "t/friends t/owesMoney"? It will be more relevant to our context.

Reply:
thanks! I am changing it.